### PR TITLE
create a new page for travel method selection

### PR DESCRIPTION
**Pull request by Amutha37**
Please view the base design for the travel method selection button. If you want to test this all you have to do is replace question 4 with this travel method component. It will work as it should be. Save button need to be change  . Main component that support this save button need editing.

### CODE REVIEW

**Comment by Amutha37** [comment]
trello link for this task : https://trello.com/c/z50v3GrA/297-dev-travel-method-selection-page

File: components/LinkButton/LinkButton.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import Link from "next/link";
-import { Box, Button, Text } from "@chakra-ui/react";
-import BackArrow from "../../public/images/back-arrow.svg";
+import Link from "next/link"
+import { Box, Button, Text } from "@chakra-ui/react"
+import BackArrow from "../../public/images/back-arrow.svg"
```
Comment (batteries76) [inline components/LinkButton/LinkButton.jsx:3]: It's not a huge problem (I'm no maniac for semi-colons), but I'm interested how they are disappearing?

Comment (Amutha37) [reply components/LinkButton/LinkButton.jsx]: This semi-colons is from my vscode setting issue. I have fixed it now. New commit is up-todate with comented lines cleaned up.

Comment (batteries76) [reply components/LinkButton/LinkButton.jsx]: Thanks Amutha, makes the changes easier to read.

Comment (batteries76) [reply components/LinkButton/LinkButton.jsx]: I guess this is another reason to have something like Prettier set up on projects.

**Pull request closed without merging**